test/leona/core_test.clj

```diff
@@ -409,3 +411,41 @@
                   (leona/compile))
         r (leona/execute schema "{ myQuery { result } }")]
     (is (= "hello"  (get-in r [:data :myQuery :result])))))
+
+;;;;;
+
+(deftest parsed-fspec-with-docstring-test
+  (s/def ::num int?)
+  (s/def ::yes true?)
+  (s/def ::result (s/keys :req-un [::num ::yes]))
+  (s/def ::args (s/keys :req-un [::num]))
+  (s/fdef res-defined :args ::args :ret ::result)
+  (defn res-defined "A docstring"
+    [ctx {:keys [num]} value]
+    {:num (* num num num) :yes true})
+
+  (let [compiled-schema (-> (leona/create)
+                            (leona/attach-query res-defined)
+                            (leona/attach-mutation res-defined)
+                            (leona/compile))
+        result (leona/execute compiled-schema "{ result(num: 3) { num }}" {:num 3} {})]
+    ; (is (= "A docstring" (get-in compiled-schema [:generated :queries :result :description])))
+    ; ^ works from within same ns, otherwise not. hence (run-tests) works, lein test fails.
```

Comment:
I'd like to understand more about why this test doesn't pass

Reply:
Symbols from other namespaces didn't make it through when juggled through. But made me discover an exception issue with fspec so, all good. Needs some more test maybe.